src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -15,7 +15,7 @@ class GradleBuilder extends AbstractBuilder {
 
   def build() {
     addVersionInfo()
-    gradle("assemble")
+    gradle("assemble", additionalOptions = "--refresh-dependencies")
```

Comment:
this seems to say that it ignores downloaded dependencies, what's the reason for this?
https://stackoverflow.com/a/31832995/4951015

If it's a one-off build agent issue the fix should be just to delete the build agent.

This will likely slow builds down a bit but probably not too badly as we are caching dependencies at artifactory

Reply:
@timja can't say I am 100% with Java, so I am taking advice from the developer who said it is needed due to dependencies not downloading correctly into the container: https://tools.hmcts.net/jira/browse/DTSPO-4836

how do we remove a build agent instead?

Reply:
just ask on platops-help and we can delete the agent.